odata-mock/README.md

```diff
@@ -5,155 +5,31 @@
 # OData Mock
 [![npm version](https://badge.fury.io/js/varkes-odata-mock.svg)](https://badge.fury.io/js/varkes-odata-mock)
 
-Mocks rest calls given an OData specification
+The OData mock mocks the application APIs based on the OData specification.
 
-After startup, you have access to:
-- odata API - http://localhost:10000/odata/
-- odata metadata - http://localhost:10000/odata/$metadata
-- rest API - http://localhost:10000/api/console
+## Features
 
-## Starting the application
+The OData mock brings you the following features:
 
-### Run local
+- Supports OData edmx specification: Copy the edmx specification to the [EDMX_MODEL_SPECIFICATION.xml](https://github.com/kyma-incubator/varkes/blob/master/examples/odata-mock-app/EDMX_MODEL_SPECIFICATION.xml) file and the Engine will automatically create data model files for the entities specified in the specification
```

Comment:
I left it, but it seems to be no longer valid (the file is not there) - I guess we should either rewrite it or remove it.

Reply:
- Provides a OData server configured with EDMX files based on https://github.com/htammen/n-odata-server

Reply:
oh, see last feature.